dockstore-webservice/src/main/resources/migrations.1.13.0.xml

```diff
@@ -26,4 +26,14 @@
             <column name="bio" type="text"/>
         </addColumn>
     </changeSet>
+    <changeSet author="ktran" id="convertUserProfileEmptyStringToNull">
+        <update tableName="user_profile">
+            <column name="link"/>
```

Comment:
To confirm, if you omit a value from the `column` tag, the value defaults to `null`?

Reply:
Yeah, I took inspiration from the [1.4.0 migrations file](https://github.com/dockstore/dockstore/blob/3b975ff8b8ed110938d046a527aecfde4a09771c/dockstore-webservice/src/main/resources/migrations.1.4.0.xml#L21)

Reply:
Cool, thanks!